source/agora/node/Runner.d

```diff
@@ -129,11 +131,20 @@ public Listeners runNode (Config config)
 
     setTimer(0.seconds, &result.node.start, Periodic.No);  // asynchronous
 
-    auto tls_ctx = getTLSContext();
+    auto tls_ctx = getTLSContext(log);
+    if (tls_ctx)
+        log.info("Node is setup to enable https");
+    else
+        log.info("Node is NOT setup to enable https");
```

Comment:
Don't think we should print anything here either. But we should error out if `result.admin !is null` and TLS isn't available.

Reply:
Will assert if admin but not configured with 
`Node is admin enabled so should have tls configured`

Reply:
Updated to throw as discussed offline.